src/java/org/apache/cassandra/io/FSError.java

```diff
@@ -38,6 +38,12 @@ public FSError(Throwable cause, Path path)
         this.file = new File(path);
     }
 
+    @Override
+    public String getMessage()
+    {
+        return String.valueOf(file);
```

Comment:
```suggestion
        return super.getMessager() + " on file " + String.valueOf(file);
```

would be nice to include `cause`'s message as before, just in case stack trace is truncated.